client_2x/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: [
+    "plugin:vue/essential",
+    "eslint:recommended",
+    "@vue/typescript/recommended",
+    "@vue/prettier",
+    "@vue/prettier/@typescript-eslint",
+  ],
+  parserOptions: {
+    ecmaVersion: 2020,
+  },
+  rules: {
```

Comment:
https://github.com/studentenhuisDS4/dsnake4/blob/develop/dsnake4-phaser/.eslintrc.js

Reply:
Vue typescript is deprecated as far as I'm aware.

Reply:
Got rid of it, slow on the uptake this morning